test/test-segmentation-lopez-acosta-2019-tiling.jl

```diff
@@ -47,7 +47,7 @@ end
     @test 0.62 ≤ round(precision; digits=2)
     @test 0.73 ≤ round(F_score; digits=2)
 end
-
+# Segmentation fault with case 102???
```

Comment:
Does this happen reliably? Can we split that singe case out as a potentially problematic broken case? Is it only in Julia 1.12?

Reply:
I was getting fairly frequent segmentation faults running on Julia 1.12, but it doesn't seem to be happening with Julia 1.11.